docs/conf.py

```diff
@@ -59,8 +53,16 @@
 sunpy_version = Version(__version__)
 is_release = not(sunpy_version.is_prerelease or sunpy_version.is_devrelease)
 
+# We want to ignore all warnings in a release version.
+if is_release:
+    warnings.simplefilter("ignore")
+warnings.filterwarnings("error", category=SunpyDeprecationWarning)
```

Comment:
I wonder if I want this to be in an else block.

Reply:
errr, I would say no, just because we really don't want to let a release out which is throwing our own deprecation warnings.